go/db/sqlite.go

```diff
@@ -0,0 +1,109 @@
+package db
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type ItemsList struct {
+	Items []ItemInfo `json:"items"`
+}
+
+type ItemInfo struct {
+	Name          string `json:"name"`
+	CategoryName  string `json:"category_name"`
+	ImageFilename string `json:"image_filename"`
+}
+
+func OpenDb(dbType string, filePath string) (db *sql.DB, err error) {
+	db, err = sql.Open(dbType, filePath)
+	if err != nil {
+		fmt.Printf("error opening database file: %v\n", err)
+	}
+	return db, err
+}
+
+func AddItemToDb(item *ItemInfo, db *sql.DB) error {
```

Comment:
There is also another approach like we can do it without passing the db param explicitly:

We can define the following structure:
```
type ItemsDB struct {
	db           *sql.DB
}
```

And define all the function in the following way:
```
func (s *ItemsDB) AddItemToDb(item *ItemInfo) error {
...
}
```

In this case we would have to initilize the structure single time (in the `main` function) and call its functions.
Also in this case the db initialization logic that I've mentioned [here](https://github.com/Vermouth347/mercari-build-training-2023/pull/3/files#r1195822649) can be executed during the initilization of this structure. 
It makes application more coupled but on the other hand we can get benefits of OOP (object oriented programming) approach